crates/iceberg/src/io/object_cache.rs

```diff
@@ -118,6 +123,20 @@ impl ObjectCache {
         }
     }
 
+    /// Set a [`ManifestList`] to the cache, if duplicate key already exist, new value will be ignored.
+    /// No-op if cache disabled.
+    pub(crate) async fn set_manifest_list(
```

Comment:
naming 
```suggestion
    pub(crate) async fn cache_manifest_list(
```
nit: name bike-shedding